### DISCUSSION TITLE
error: script "dev" exited with code 11 (SIGSEGV) when nuxt-security included using Bun

### DISCUSSION THREAD

**Discussion by kiranimeid** [Baroshem/nuxt-security#299] [Q&A] [answered]
I got this error while using nuxt-security and bun.

`error: script "dev" exited with code 11 (SIGSEGV)`

i am using wsl with bun 1.0.13 and nuxt 3.8.1 with nuxt-security 1.0.0-rc.4

**Comment by Baroshem** [comment, marked as answer]
Hey, 

Check out this issue https://github.com/Baroshem/nuxt-security/issues/294
